sparkles/core.py

```diff
@@ -336,7 +336,7 @@ def _run_aca_review(
                 aca.add_message(
                     "info",
                     text=f"Using dyn_bgd_n_faint={dyn_bgd_n_faint} "
-                         f"(call_args val={aca.dyn_bgd_n_faint})",
+                    f"(call_args val={aca.dyn_bgd_n_faint})",
```

Comment:
I'm surprised black does this, but I'd suggest instead:
```
                    text=(
                        f"Using dyn_bgd_n_faint={dyn_bgd_n_faint} "
                        f"(call_args val={aca.dyn_bgd_n_faint})"
                    ),
```

Reply:
I think black didn't do the multi-line (I did) and only wanted different spacing but if you prefer it in parens that's fine.